src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,132 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int START_LENGTH = 10;
+    private static final double LENGTH_INCREASING = 1.5;
+    private int index;
+    private T[] ourArrayList;
+
+    public ArrayList() {
+        ourArrayList = (T[]) new Object[START_LENGTH];
+        index = 0;
+    }
```

Comment:
```suggestion
        ourArrayList = (T[]) new Object[START_LENGTH];
    }
```

It will be initialized by 0 by default